assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Idea.js

```diff
@@ -186,6 +191,10 @@ export default function Idea( props ) {
 									activity === IDEA_HUB_ACTIVITY_IS_PROCESSING
 								}
 								icon={ <UnpinIcon /> }
+								title={ __(
+									'Remove from saved"',
```

Comment:
Oops!
```suggestion
									'Remove from saved',
```

Reply:
![i-hang-my-head-in-shame](https://user-images.githubusercontent.com/6187401/131889244-1d581190-dacc-4312-a825-79e821932c6d.jpg)